dev/ory/oathkeeper_rules.yaml

```diff
@@ -10,6 +10,22 @@
     handler: allow
   mutators:
     - handler: noop
+
+- id: allow-clear-cookies-endpoint
+  upstream:
+    url: "http://e2e-tests:4012"
+  match:
+    url: "<(http|https)>://<[a-zA-Z0-9-.:]+>/auth/clearCookies"
+    methods: ["GET", "POST", "OPTIONS"]
+  authenticators:
+    - handler: anonymous
+  authorizer:
+    handler: allow
+  mutators:
+    - handler: delete_cookies
```

Comment:
where is there a documentation for this one?

can't find anything here: https://www.ory.sh/docs/oathkeeper/pipeline/mutator#cookie-configuration

Reply:
this settings doesn't sound right to me. I think if the goal is to have `cookie`, it is meant to inject a cookie from the authenticator. ie: for rewriting a header to a cookie.

I don't see any `delete_cookies` mutator

Reply:
got it from chatgpt 
<img width="811" alt="Screen Shot 2023-01-17 at 9 50 32 PM" src="https://user-images.githubusercontent.com/1273575/213079167-3e7ccf6c-d50b-46f7-b08c-337909521b4b.png">